DRIVERS.md

```diff
@@ -0,0 +1,31 @@
+# Driver plugin installation
+
+Minikube uses Docker Machine to manage the Kubernetes VM so it benefits from the
+driver plugin architecture that Docker Machine uses to provide a consistent way to
+manage various VM providers. Minikube embeds VirtualBox and VMware Fusion drivers
+so there are no additional steps to use them. However, other drivers require an
+extra binary to be present in the host PATH.
+
+The following drivers currently require driver plugin binaries to be present in
+the host PATH:
+
+* [KVM](#kvm-driver)
```

Comment:
As discussed in other issues, that is the goal but there some outstanding problems with the xhyve driver atm. #262 adds embedded KVM driver for Linux & the same technique can be used for xhyve once we've sorted the driver issues.